STM32Cube/Tasks/can_handler.c

```diff
@@ -32,6 +34,15 @@ void can_handle_rx(const can_msg_t *message, uint32_t timestamp) {
 	    data.time = millis_();
 	    result = xQueueOverwriteFromISR(altQueue, &data, &xHigherPriorityTaskWoken);
 	    break;
+	case MSG_STATE_EST_CALIB:
+		uint8_t flag; //sent to processor low
+		uint16_t apogee;
+		if(get_state_est_calibration_msg(message, &flag, &apogee) && !flag) //left side is guaranteed to evaluate before right side
+		{
+			result |= xQueueSendFromISR(targetQueue, &apogee, &xHigherPriorityTaskWoken); //send new target apogee to controller
+			result |= xEventGroupSetBitsFromISR(calibrationEventHandle, RESET_FILTER_FLAG, &xHigherPriorityTaskWoken); //tell state estimation to reset filter algorithm
+		}
+		break;
 
 	default:
```

Comment:
completely unrelated to this pr, but is receiving an unknown CAN msg ever possible? for example should this `default:` case log a warnin
 idrk how CAN works ngl

Reply:
We receive every CAN message sent over the bus, period. So this case handles every type of CAN message we don't explicitly care about like say the CC pressure.